public/assets/svg/bell.tsx

```
@@ -1,19 +1,19 @@
-const IconBell = ({ size = 20, color = '#A1A1A1', ...props })  => (
+const IconBell = ({ size = 20, color = '#A1A1A1', ...props }) => (
   <svg
-    xmlns="http://www.w3.org/2000/svg"
+    xmlns='http://www.w3.org/2000/svg'
     width={size}
     height={size}
-    viewBox="0 0 20 20"
+    viewBox='0 0 20 20'
     fill={color}
     {...props}
   >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)